Fitter/scripts/overlay_operator_variations_wrapper.py

```diff
@@ -59,23 +59,34 @@
 ]
 
 for d in dirs:
-    print d
-    input_dir  = os.path.join(CONST.EFTFIT_TEST_DIR,dir_ver,d)
-    output_dir = os.path.join(web_area,'eft_stuff/misc/fitting_plots',dir_ver,"{tstamp}_{dir}".format(tstamp=TSTAMP2,dir=d))
+    #print d
+    input_dir  = os.getcwd()#os.path.join(CONST.EFTFIT_TEST_DIR,dir_ver,d)
```

Comment:
I know this is sort of a case of "six of one, half a dozen of the other", but is there a reason you changed this?

The entire point of this script was to generate plots by reading inputs (generally `root` files) from a specified location and then produce those plots and have them moved automatically to another location, while also applying some additional post-processing where needed.

I might suggest refactoring this script by having most of these per-user options specified via command line options.